tests/routes.users.js

```diff
@@ -0,0 +1,96 @@
+const assert = require('assert');
+const httpStatus = require('http-status-codes');
+const supertest = require('supertest');
+const models = require('../models')
+const app = require('../app');
+const User = require('../models').User
+
+describe('GET /api/users', () => {
+    it('return all users', async () => {
+        const testUser = {
+            user: {
+                firstName: 'testName1',
+                age: '25'
+            }
+        };
+        await User.create(testUser.user);
+
+        const res = await supertest(app)
+            .get('/api/users')
+            .expect(httpStatus.OK);
+
+        if (res.body.user.id) {
```

Comment:
You don't need to have such if constructions in the tests. 
In this test you should get the array of users here and check that you returned all the existing users by comparing total users count in the database